scss/_variables.scss

```diff
@@ -528,6 +528,7 @@ $zindex-tooltip:            1070 !default;
 
 // Navs
 
+$nav-link-padding:              .5rem !default;
```

Comment:
I have made your requested changes, but for navbar link variable i have splited  in left/right instead of x/y as this are for left and right side not left/right and top/bottom.